app/dashboard/migrations/0101_auto_20200420_1427.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 2.2.4 on 2020-04-20 14:27
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
Im ok with this but @danlipert told about fields removals the other day and kind of make sense, anyway guys I let to you decide on this removal or not.

Reply:
we've got the data backed up! 
And we are shutting down the discord server this week !